server/models/annotation.js

```diff
@@ -43,13 +46,14 @@ annotationSchema.pre('save', function preSave(next) {
 
   const fillReply = new Promise((resolve, reject) => {
     if (this.parent) {
-      this.constructor.findByIdAndUpdate(this.parent, { $inc: { numChildren: 1 } })
+      this.constructor.findByIdAndUpdate(this.parent, { $inc: { numChildren: 1 }, $push: { childAnnotations: this._id } }) // @angela: here is where we add childAnnotations
       .then((parent) => {
         this.article = parent.article;
         this.articleText = parent.articleText;
         this.ranges = parent.ranges;
         this.isPublic = parent.isPublic;
         this.groups = parent.groups;
+        this.isTopLevel = false; // @angela: maybe we don't need to do this , see virtual?
```

Comment:
I tried for fun and got rid of this, replacing it with virtual column -- should test